gamestonk_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -165,6 +189,275 @@ def call_ayr(self, other_args: List[str]):
         if ns_parser:
             terraengineer_view.display_anchor_yield_reserve(export=ns_parser.export)
 
+    def call_sinfo(self, other_args: List[str]):
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="sinfo",
+            description="""
+                Displays staking info of a certain terra address.
+                [Source: https://fcd.terra.dev/swagger]
+            """,
+        )
+        parser.add_argument(
+            "-a",
+            "--address",
+            dest="address",
+            type=str,
+            help="Terra address. Valid terra addresses start with 'terra'",
+            required="-h" not in other_args,
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            dest="limit",
+            type=check_positive,
+            help="Number of delegations",
+            default=10,
+        )
+
+        if other_args and not other_args[0][0] == "-":
+            other_args.insert(0, "-a")
+
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        if ns_parser:
+            terramoney_fcd_view.display_account_staking_info(
+                export=ns_parser.export, address=ns_parser.address, top=ns_parser.limit
+            )
+
+    def call_validators(self, other_args: List[str]):
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="validators",
+            description="""
+                Displays information about terra validators.
+                [Source: https://fcd.terra.dev/swagger]
+            """,
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            dest="limit",
+            type=check_positive,
+            help="Number of validators to show",
+            default=10,
+        )
+        parser.add_argument(
+            "-s",
+            "--sort",
+            dest="sortby",
+            type=str,
+            help="Sort by given column. Default: votingPower",
+            default="votingPower",
+            choices=[
+                "validatorName",
+                "tokensAmount",
+                "votingPower",
+                "commissionRate",
+                "status",
+                "uptime",
+            ],
+        )
+
+        parser.add_argument(
```

Comment:
i think `--descend` argument is not working
![image](https://user-images.githubusercontent.com/43375532/149749857-be884a38-b1f8-4124-ad1d-fe9ed5a875e7.png)